runtime/seq.go

```diff
@@ -173,6 +173,24 @@ func seqFindFirst(f *Frame, iterable *Object, pred func(*Object) (bool, *BaseExc
 	return false, nil
 }
 
+func seqFindElem(f *Frame, elems []*Object, o *Object, start, stop int) (int, bool, *BaseException) {
```

Comment:
Rather than passing start and stop, I would have the caller slice the elems: `seqFindElems(f, elems[start:stop], o)`